suite-native/module-accounts/src/components/TransactionListHeader.tsx

```diff
@@ -31,38 +32,48 @@ type AccountsNavigationProps = TabToStackCompositeNavigationProp<
     AccountsStackParamList
 >;
 
-export const TransactionListHeader = memo(({ accountKey }: AccountDetailHeaderProps) => {
-    const navigation = useNavigation<AccountsNavigationProps>();
-    const accountHasTransactions = useSelector((state: AccountsRootState) =>
-        selectHasAccountTransactions(state, accountKey),
-    );
-    const isTestnetAccount = useSelector((state: AccountsRootState) =>
-        selectIsTestnetAccount(state, accountKey),
-    );
+export const TransactionListHeader = memo(
+    ({ accountKey, tokenSymbol }: AccountDetailHeaderProps) => {
+        const navigation = useNavigation<AccountsNavigationProps>();
+        const accountHasTransactions = useSelector((state: AccountsRootState) =>
+            selectHasAccountTransactions(state, accountKey),
+        );
+        const isTestnetAccount = useSelector((state: AccountsRootState) =>
+            selectIsTestnetAccount(state, accountKey),
+        );
 
-    const handleReceive = () => {
-        navigation.navigate(AppTabsRoutes.SendReceiveStack, {
-            screen: SendReceiveStackRoutes.Receive,
-            params: { accountKey },
-        });
-    };
-    return (
-        <>
-            <AccountDetailGraphHeader accountKey={accountKey} />
-            {accountHasTransactions && (
-                <>
-                    {!isTestnetAccount && <AccountDetailGraph accountKey={accountKey} />}
-                    <Box marginBottom="large" paddingHorizontal="medium">
-                        <Button iconLeft="receive" size="large" onPress={handleReceive}>
-                            Receive
-                        </Button>
-                    </Box>
-                    <Divider />
-                </>
-            )}
-            <Box marginTop="extraLarge" marginBottom="medium" marginHorizontal="large">
-                <Text variant="titleSmall">Transactions</Text>
-            </Box>
-        </>
-    );
-});
+        const handleReceive = () => {
+            navigation.navigate(AppTabsRoutes.SendReceiveStack, {
+                screen: SendReceiveStackRoutes.Receive,
+                params: { accountKey },
+            });
+        };
+        return (
+            <>
+                {/* Graph is temporarily hidden for ERC20 tokens. */}
+                {/* Will be solved in issue: https://github.com/trezor/trezor-suite/issues/7839 */}
+                {!tokenSymbol && (
```

Comment:
extra variable `isToken`

Reply:
fixed in https://github.com/trezor/trezor-suite/commit/4d5c05278740cdd3d9e85981c1257cf8feac0a25